modules/installation-creating-aws-vpc-subnets-edge.adoc

```diff
@@ -1,12 +1,17 @@
 // Module included in the following assemblies:
 //
-// * post_installation_configuration//aws-compute-edge-zone-tasks..adoc (AWS Local Zone or Wavelength Zone tasks) 
+// * post_i// * post_installation_configuration/configuring-aws-outposts.adoc
+
+ifeval::["{context}" == "configuring-aws-outposts"]
+:outposts:
+endif::[]
 
 :_mod-docs-content-type: PROCEDURE
 [id="installation-creating-aws-vpc-subnets-edge_{context}"]
-= Creating subnets in AWS Local Zones or Wavelength Zones
+= Creating subnets for AWS edge compute services
 
-Before you configure a machine set for edge compute nodes in your {product-title} cluster, you must create the subnets in {zone-type}. Complete the following procedure for each Wavelength Zone that you want to deploy compute nodes to.
+Before you configure a machine set for edge compute nodes in your {product-title} cluster, you must create a {zone-type} subnet.
```

Comment:
The current phrasing renders as "you must create a AWS Outposts subnet."
Possible suggestion to avoid this:
```suggestion
Before you configure a machine set for edge compute nodes in your {product-title} cluster, you must create the {zone-type} subnet.
```
or

```suggestion
Before you configure a machine set for edge compute nodes in your {product-title} cluster, you must create a subnet in {zone-type}.
```